### ISSUE TITLE
Accessibility - combobox violates aria 1.2 rules

### ISSUE THREAD

**Issue by kyjak** [carbon-design-system/carbon-for-ibm-dotcom#6117]
The combobox design pattern is detected as WAI-ARIA 1.1, which is not allowed by WAI-ARIA 1.2
reporting url https://ibmdocs-dev.mybluemix.net/docs/en/error
you can use the ibm equal access checker

`<div aria-label="Search IBM Documentation" class="react-autosuggest__container" aria-expanded="false" aria-owns="react-autowhatever-1" aria-haspopup="listbox" role="combobox">`

**Comment by RobertaJHahn** [comment]
Combo box is a Core Carbon component.  Kenny will review and document more details for the next review.  D.A. Kahn is the Carbon accessibility contact.
